packages/sdk/src/actions/relayer.ts

```diff
@@ -0,0 +1,99 @@
+import axios from "axios";
+import { getTokenStorage } from "./ticket";
+import { getTransaction } from "../helpers";
+import { sendTransactionsCallback } from "./connection";
+import type {
+  ConnectionType,
+  PublicArgsInterface,
+  RelayerDataInterface,
+} from "../interfaces";
+import { OneYOctoNear } from "../constants";
+
+export const sendWithdraw = async (
+  relayer: RelayerDataInterface,
+  payload: { publicArgs: PublicArgsInterface; currencyAccountId: string }
```

Comment:
currencyAccountId is not part of the relayer interface (in relayer code) - how is the specification of the currency happening?

Reply:
From what I see, the payload interface in the relayer is this:
```
  const {
    payload: { tokenId, price_token_fee, reciver_storage, currencyContractId },
  } = jwt.decode(token);
```

Therefore when withdrawing the `currencyAccountId ` parameter would not be necessary as it was already passed in the fee query